### PR TITLE
[ADF-5331] - Alpha ADF/JS-API update has been successfully created.

### PR DESCRIPTION
**Pull request by eromano**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

In order to improve the ADF/JS-API Alpha build we have decided to change :

The name of the Alpha build will contain the build number and not the hash

The JS-API will be updated by an automatic task run by the JS-API at its end

ADF will not automatically update anymore the JS-API

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: scripts/next_version.sh [hunk 1]
```diff
@@ -78,8 +78,7 @@ fi
 
 if [[ "${EXEC_ALPHA}" == true ]]
 then
-    ISH_KEY=$(git rev-parse HEAD)
-    NEXT_VERSION=${NEXT_VERSION}-${ISH_KEY}
+    NEXT_VERSION=${NEXT_VERSION}-${TRAVIS_BUILD_NUMBER}
```
Comment (popovicsandras) [inline scripts/next_version.sh:81]: Is this script only used in the CI? `TRAVIS_BUILD_NUMBER` is provided only in CI, if it is used only in the CI, we might consider to move it under `scripts/travis`, so no one can have the false expectation to work locally.

Comment (popovicsandras) [reply scripts/next_version.sh:81]: If not, I suggest this `TRAVIS_BUILD_NUMBER` to have some more generic name, and be an input param for this script.

File: scripts/check-js-api-version.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+require('dotenv').config({path: process.env.ENV_FILE});
+const fs = require('fs');
+
+const configPath = './dist/demo-shell/app.config.json';
+
+fs.readFile(configPath, (err, appConfigString) => {
```
Comment (popovicsandras) [inline scripts/check-js-api-version.js:9]: Is it going to wait before ending the script?? Should not it be readFileSync, or some Promise/async-await magic?

**Review by popovicsandras — APPROVED** [review]

File: scripts/next_version.sh (hunk 1 above)
Comment (eromano) [reply scripts/next_version.sh:81]: only CI

**Pull request merged**